statsd/src/test/java/com/avast/metrics/statsd/StatsDMeterTest.java

```diff
@@ -0,0 +1,28 @@
+package com.avast.metrics.statsd;
+
+import com.timgroup.statsd.StatsDClient;
+import org.junit.Test;
+import org.mockito.Matchers;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class StatsDMeterTest {
+
+    @Test
+    public void testCounts() {
```

Comment:
I like separating lines (or groupping lines) by context. However I agree it didn't end well in this case :-)